sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachinePopularImageTests.java

```diff
@@ -37,34 +37,36 @@ public void canCreateAllPopularImageVM() {
             .collect(Collectors.toList())) {
             Mono<VirtualMachine> mono = computeManager.virtualMachines()
                 .define(generateRandomResourceName("vm", 10))
-                .withRegion(Region.US_SOUTH_CENTRAL)
+                .withRegion(Region.US_WEST2)
                 .withNewResourceGroup(rgName)
                 .withNewPrimaryNetwork("10.0.0.0/24")
                 .withPrimaryPrivateIPAddressDynamic()
                 .withoutPrimaryPublicIPAddress()
                 .withPopularWindowsImage(image)
                 .withAdminUsername("testUser")
                 .withAdminPassword(password())
-                .withSize(VirtualMachineSizeTypes.fromString("Standard_D2a_v4"))
+                .withSize(VirtualMachineSizeTypes.STANDARD_D2S_V3)
                 .createAsync();
             vmMonos.add(mono);
         }
 
         for (KnownLinuxVirtualMachineImage image : Arrays.stream(KnownLinuxVirtualMachineImage.values())
-            .filter(image -> image != KnownLinuxVirtualMachineImage.OPENSUSE_LEAP_15_1 && image != KnownLinuxVirtualMachineImage.SLES_15_SP1)
+            .filter(image -> image != KnownLinuxVirtualMachineImage.OPENSUSE_LEAP_15_1
+                && image != KnownLinuxVirtualMachineImage.SLES_15_SP1
+                && image != KnownLinuxVirtualMachineImage.ORACLE_LINUX_8_1)
```

Comment:
Is there a new version for oracle linux?

We may want to add deprecation message to `ORACLE_LINUX_8_1`, and add a newer version of it.

Reply:
The `ORACLE_LINUX_8_1` seems to be deprecated.
The detail information following as:
[test.log](https://github.com/user-attachments/files/16689115/test.log)
[compute-test.log](https://github.com/user-attachments/files/16689127/compute-test.log)